docs/schema.md

```diff
@@ -96,15 +97,16 @@ All properties marked as "not undefined" will be present in the end record, the
    */
```

Comment:
What does that mean? Aren't the attributes after this comment in the final record?

If not, we should remove them from the docs and add them back only when they are.

Reply:
We could just comment the markdown or remove them.
Not sure when we will expose the multi record strategy

Reply:
So everything after the comment can go?

Reply:
Removed in 153994166c142b86e365d10a787afe91538e652b.